src/DynamoRevit/Models/RevitDynamoModel.cs

```diff
@@ -355,12 +349,16 @@ private void SubscribeApplicationEvents(ExternalCommandData commandData)
                 return;
             }
 
-            commandData.Application.ViewActivating += OnApplicationViewActivating;
-            commandData.Application.ViewActivated += OnApplicationViewActivated;
+            proxy = new EventHandlerProxy<RevitDynamoModel>(this);
```

Comment:
It  has no issues if SubscribeApplicationEvents happens in the scope of the execute method of the command. But I think it is safer to put it in an idling event.